client/elife-theme/src/components/index.js

```diff
@@ -0,0 +1 @@
+export { default as Button } from './Button'
```

Comment:
Remind me why you think that index files like this are useful for private components?

Reply:
The first time I used something like this it seemed more maintainable in the theme `index.js` to import/remove imported components (e.g. instead of having a lot of different imports, you can only do one). `xpub-epmc` is doing the same thing, so it might be nice to have a similar style https://gitlab.coko.foundation/xpub/xpub-epmc/tree/dev/app/epmc-theme/elements